harvardcards/static/js/modules/collections-index.js

```diff
@@ -1,11 +1,30 @@
-define(['jquery', 'components/slider/CollectionSlider', 'utils/utils'], function($, CollectionSlider, utils) {
+define(['jquery', 'components/slider/CollectionSlider', 'utils/utils', 'jqueryui'], function($, CollectionSlider, utils) {
 	return {
 		initModule: function(){
-			$('.slider').each(function(index, el) {
-				var slider = new CollectionSlider(el);
-			});
 
-			utils.setupConfirm();
+
+            $('.courseHeader').click(function() {
+                var el = this;
+                $(this).next().toggle('slow', function(){
+                    var collection_id = $(el).data('collection-id');
+                    if($(el).next().is(":visible")){
+                        localStorage.setItem('show-collection-id'+collection_id, true);
+                    } else {
+                        localStorage.removeItem('show-collection-id'+collection_id);
+                    }
+                });
+                $(this).find('.plusminus').toggleClass("fa-plus-circle fa-minus-circle");
+                return false;
+            }).next().hide();
+
+            $('.courseHeader').each(function(index){
+                var el = this;
+                var collection_id = $(el).data('collection-id');
+                if(localStorage.getItem('show-collection-id'+collection_id) == 'true'){
+                    $(el).click();
```

Comment:
This would probably be better if we could call the function directly that manipulates the expand/collapse behavior, rather than indirectly triggering a "click".